src/module_ncio.f90

```diff
@@ -5,52 +5,77 @@
 !! variables and char variables. Variables can have up to 5 dimensions.
 !! @author jeff whitaker <jeffrey.s.whitaker@noaa.gov> @date 201910
 module module_ncio
-
+!
   use netcdf
   use mpi
-
+!
   implicit none
   private
-
+!
   type Variable
-     integer varid ! netCDF variable ID
-     integer ndims ! number of dimensions
-     integer dtype ! netCDF data type
-     integer natts ! number of attributes
-     integer deflate_level ! compression level (if > 0)
-     logical shuffle  ! shuffle filter?
-     logical hasunlim ! has an unlimited dim?
-     character(len=nf90_max_name) :: name ! variable name
-     integer, allocatable, dimension(:) :: dimids ! netCDF dimension IDs
-     ! indices into Dataset%dimensions for associated dimensions.
+! netCDF variable ID
```

Comment:
If these comments are doxygen comments, they go on the same line as the variable, with the special comment marker '!<'.

Like this:
```
       integer varid !< NetCDF variable ID.
       integer ndims !< Number of dimensions.

```